eZ/Bundle/PlatformInstallerBundle/src/Resources/config/database.yml

```diff
@@ -0,0 +1,18 @@
+services:
+    ezplatform.installer.database.connection.factory:
+        class: EzSystems\PlatformInstallerBundle\Database\ConnectionFactory
+        arguments: ["@ezpublish.persistence.connection"]
+
+    ezplatform.installer.database.connection:
+        class: Doctrine\DBAL\Connection
+        factory: ["@ezplatform.installer.database.connection.factory", createConnection]
+
+    ezplatform.installer.database.platform.mysql:
```

Comment:
@bdunogier Doctrine\DBAL refers to the strategy/class for a specific driver (MySQL, PostgreSQL, etc.) as [DatabasePlatform](http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/platforms.html) and defines `AbstractPlatform` to inherit from.